Location permission on Map

Vanethos opened the pull request:
Add location permissions to map screen

Vanethos committed:
Add location_permissions dialog when going to maps screen

Vanethos committed:
Remove unneccessary import

miguelpruivo commented on lib/presentation/ui/home/home_page.dart:
```diff
@@ -264,7 +292,55 @@ class MapWidget extends StatelessWidget {
+            alignment: Alignment(0, 0),
+            width: size.shortestSide * .7,
+            child: Text(
+              "This app needs the GPS location so that it can show where you are in this map, do you want to allow it?",
```
Shouldn't this be on internalization support?

miguelpruivo commented on lib/presentation/ui/home/home_page.dart:
```diff
@@ -264,7 +292,55 @@ class MapWidget extends StatelessWidget {
+              Navigator.of(context).pop();
+              },
+            child: Text(
+              "Allow Location".toUpperCase(),
```
The same.

miguelpruivo commented on lib/presentation/ui/home/home_page.dart:
```diff
@@ -264,7 +292,55 @@ class MapWidget extends StatelessWidget {
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: <Widget>[
+        Container(child: Text("Permissions", style: styleIntroTitle(),)),
```
Shouldn't this be on internalization support?

Vanethos commented:
Regarding the comment: you are right, but since we don't have yet the copy, I thought of opening an issue in the future about it

Vanethos committed:
Add permission details to iOS info.plist

Merge branch 'develop' into feature/VOST-85-location-permission
# Conflicts:
#	lib/presentation/assets/dimensions.dart
#	lib/presentation/ui/home/home_page.dart

Commit:
Merge branch 'develop' into feature/VOST-85-location-permission

miguelpruivo reviewed (approved):
Internalization support will be added on the future.
